app/models/user.rb

```diff
@@ -1,6 +1,10 @@
 class User < ActiveRecord::Base
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable,
+         :recoverable, :rememberable, :validatable
   validates :first_name, :last_name, :locale, presence: true
-  devise :database_authenticatable, :registerable,
+  devise :database_authenticatable,
```

Comment:
We have duplicated these lines, keep only one:
```rb
  devise :database_authenticatable, :recoverable, :rememberable, :validatable
```

Reply:
Done! :+1: